chatmodel-springai/src/test/java/com/example/ai/controller/ChatControllerTest.java

```diff
@@ -76,6 +78,26 @@ void sentimentAnalyzer() {
                 .body("answer", is("SARCASTIC"));
     }
 
+    @Test
```

Comment:
@coderabbitai, please review all test cases in this class and provide if there are any suggestions or improvements

Reply:
@coderabbitai, can you please help me with writing code to implement all above suggestions.